post_installation_configuration/cluster-tasks.adoc

```diff
@@ -197,9 +197,9 @@ For clusters managed by the Assisted Installer, you can add worker nodes by usin
 
 * xref:../nodes/nodes/nodes-sno-worker-nodes.adoc#sno-adding-worker-nodes-to-single-node-clusters-manually_add-workers[Manually adding worker nodes to a SNO cluster]
 
-=== Adding worker nodes to clusters managed by the multicluster engine for Kubernetes
+=== Adding worker nodes to clusters managed by the {mce}
```

Comment:
Is "managed by the multicluster engine for Kubernetes operator" the same as "managed by the multicluster engine for Kubernetes"?

Reply:
Reverted the change. You are right it's better to not say "Operator" here.